pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1085,3 +1118,10 @@ fn define_inventory_class(inventory_class_name: &syn::Ident) -> TokenStream {
         _pyo3::inventory::collect!(#inventory_class_name);
     }
 }
+
+const DUPE_SET: &str = "duplicate `set` - the struct is already annotated with `set_all`";
+const DUPE_GET: &str = "duplicate `get` - the struct is already annotated with `get_all`";
```

Comment:
Nit: I think I'd use "useless" or "redundant" rather than duplicate?

```suggestion
const DUPE_SET: &str = "useless `set` - the struct is already annotated with `set_all`";
const DUPE_GET: &str = "useless `get` - the struct is already annotated with `get_all`";
```